src/commands/keywords.rs

```diff
@@ -114,11 +110,11 @@ pub async fn add(
 		Some(captures) => {
 			let keyword = captures
 				.get(1)
-				.ok_or("Captures didn't contain keyword")?
+				.ok_or_else(|| anyhow!("Captures didn't contain keyword"))?
```

Comment:
[The function will always be called and potentially allocate an object acting as the default.](https://rust-lang.github.io/rust-clippy/master/index.html#or_fun_call)

Reply:
Can these use `.context("message")` instead? [That method](https://docs.rs/anyhow/1.0.37/src/anyhow/context.rs.html#83-88) is essentially equivalent to this.

Reply:
Sure